### DISCUSSION TITLE
Putty Fatal Error : Network error : Connection time out

### DISCUSSION THREAD

**Discussion by Chandanag24** [learndevopsonline/learndevopsonline#1635] [Q&A] [answered]
![Screenshot (2)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/f7570d97-2188-409f-869d-523863867f09)

**Comment by sreedharm07** [comment]
@Chandanag24 
Check security group attached to instance 
Security group should be allow all

**Reply by Chandanag24** [reply]
![Screenshot (3)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/e6de4089-674d-4335-a0b4-37c98ef4a15e)
I have given security group as allow-all only. previously it is working fine but now i am facing an issue like fatal issue like that..
![Screenshot (4)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/380323d5-b1f4-44f6-b26f-3e7063a311fc)

**Reply by sreedharm07** [reply]
Wait some time and try it will login 

Instance will take time to come up

**Reply by sreedharm07** [reply]
Check 
- centos8 ami 
- no keypair
- security group with allow all 
Please check all this check list

**Reply by Chandanag24** [reply]
Yes i tried and waited for it almost one week but same error i am facing...
Ok.I will check all this list as you mentioned above and let you know..

**Reply by Chandanag24** [reply]
![Screenshot (5)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/d403d8b1-a82d-4f98-a80f-ed4d60403bb8)
![Screenshot (6)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/276e9753-2531-43b1-b413-b25fd89058c7)
I have given all the check list correct only...just now i checked all the things..

**Reply by sreedharm07** [reply]
I see all confiration is correct.
Delete that instance and 
Try to create again

**Reply by Chandanag24** [reply]
Ok.Sure.Thanks.

**Reply by sreedharm07** [reply, marked as answer]
@Chandanag24 
It seems there is no internet gateway
May be you removed internet gateway 
Please check it and if you removed please configure it

**Reply by Chandanag24** [reply]
Hi, @sreedharm07 
Here, it is my new instance i have created as workstationn this new one also not connecting....same issue i am facing..I tried it couple of times but i didn't.. below are my instance details...
![Screenshot (7)](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/31d839a3-c6fa-4da1-b769-de9c7f9b19c6)
![1](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/9b07a5bd-010b-43e0-aea2-b82c93adeac6)
![2](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/9f998e1a-2099-4ce1-9cc1-5be8f2b26ad2)
![3](https://github.com/learndevopsonline/learndevopsonline/assets/140817248/5eecd9f6-c1df-4e04-bf1e-e5c27d696248)

**Reply by sreedharm07** [reply]
Its a closed discussion 
It wont be notified in further to help 
can you please raise new discussion

**Reply by Chandanag24** [reply]
Ok.Sure.I will raise a new discussion.
